hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/s3guard/DDBCapacities.java

```diff
@@ -82,7 +79,7 @@ public String toString() {
   }
 
   /**
-   * Is the the capacity that of a pay-on-demand table?
+   * Is the the capacity that of a pay-per-request table?
```

Comment:
I don't understand this change. What's the diff between on-demand and per-request?

Reply:
The AWS docs say "on demand" or "pay by request"; I was trying to be consistent but gave up. This is a leftover from an attempt/revert of actually changing the method name. Changing to "on demand" (and not pay-on-demand)